app/models/setting/discovered.rb

```diff
@@ -42,6 +48,16 @@ def self.load_defaults
       end
     end
 
+    Setting.transaction do
+      [
+          self.set('discovery_facts_highlights', N_("The default regex to organize facts for highlights section"), "" ),
+          self.set('discovery_facts_storage', N_("The default regex to organize facts for storage section"), "" ),
+          self.set('discovery_facts_software', N_("The default regex to organize facts for software section"), "" ),
+          self.set('discovery_facts_hardware', N_("The default regex to organize facts for hardware section"), "" ),
+          self.set('discovery_facts_network', N_("The default regex to organize facts for network section"), "" ),
+          self.set('discovery_facts_ipmi', N_("The default regex to organize facts for ipmi section"), "" ),
```

Comment:
Three more nitpicks:

1) Drop the `The default" from the setting name, I'd go with the short "Regex to organize..."

2) Extra space after `""`

3) Rewrite the commit message, it contains some garbage